tpe/lib/src/Model_TEST.cc

```diff
@@ -133,11 +133,11 @@ TEST(Model, BoundingBox)
 
   math::AxisAlignedBox expectedBoxLinkFrame(
       math::Vector3d(-2, -2, -2), math::Vector3d(2, 2, 2));
-  EXPECT_EQ(expectedBoxLinkFrame, linkEnt.GetBoundingBox(true));
+  EXPECT_EQ(expectedBoxLinkFrame, linkEnt.GetBoundingBox());
```

Comment:
Oh interesting, so basically since `_force` was being set to true before the dirty flag was actually not being tested?
Can you add a few lines to set bitmasks for the Collision objects here and test the OR-ing / `collideBitmaskDirty` logic as well?

Reply:
yeah that's correct, removing the force arg revealed that bounding box caching was not working. 

there is already a CollideBitmask test added in pull request #69 ([here](https://github.com/ignitionrobotics/ign-physics/pull/69/files#diff-8ab09be8e30cbdbf937d238f22285466R165)) and I've been using that to test this PR. It should cover `collideBitmaskDirty` logic.

Reply:
Ok saw it, looks great!